### DISCUSSION TITLE
How to use the spix automatic testing tool

### DISCUSSION THREAD

**Discussion by liushisong** [faaxm/spix#77] [General]
Hello, I want to ask why I put the instance code into the QT project. The cpp file is missing the "spix_export. h" header file, and cannot recognize many class names and function names, such as TestServer, override, wait, getStringProperty, QtQmlBot, etc.

**Comment by faaxm** [comment]
I am not sure I understand the issue. If you are getting compiler/linker errors with your project, have you tried compiling the examples that come with spix?
